src/main/java/com/esri/samples/displayinformation/sketch_on_map/README.md

```diff
@@ -0,0 +1,47 @@
+<h1>Sketch on Map</h1>
+
+<p>Use the Sketch Editor to edit, or sketch a new point, line, or polygon geometry on to a map. </p>
+
+<p><img src="SketchOnMap.png"/></p>
```

Comment:
Very cool picture!

Reply:
Thanks! It's a good tool to highlight the previous advances of the glacier :)